### ISSUE TITLE
Missing Twister Coaster Piece

### ISSUE THREAD

**Issue by raxxxor18** [OpenRCT2/OpenRCT2#3361]
**OS:**
Windows 10 64bit
**Version:** 
Every Version I tried from 0.0.4 - 0.0.5

There is a track piece of the twister roller coaster that is not buildable in openrct. Imported coasters that have this piece built in work flawlessly, but there is no option to build the piece with the editor.

It's the small curve that goes from no roll slope down to a rolled level track. Interestingly the counterpart (rolled level to slope up no roll) is buildable.

**Steps to reproduce:**
1. Build a no roll slope down piece in the twister coaster
2. See that the "level" icon is greyed out. 

Interesting occurance: If you build a rolled piece backwards, the button for the track is not greyed out and clickable, but the build track button goes empty and unclickable. 

**Screenshots / Video:**
Vanilla RCT
![rct](https://cloud.githubusercontent.com/assets/2523938/14690781/188136ac-074e-11e6-8c84-a4f27fc6dde3.png)
OpenRCT
![rct2](https://cloud.githubusercontent.com/assets/2523938/14690789/2828bdd2-074e-11e6-87ef-05b7a032b765.png)

**Comment by marijnvdwerf** [comment]
Ah, so that's the reason I couldn't find them, despite there being descriptions for them in #2967.

```
// Fully known, but not yet ecountered
{   false,  TRACK_SLOPE_DOWN_25,    TRACK_BANK_NONE,    TRACK_CURVE_LEFT_SMALL,         TRACK_SLOPE_NONE,       TRACK_BANK_LEFT,    TRACK_ELEM_LEFT_QUARTER_TURN_3_TILES_25_DEG_DOWN_TO_LEFT_BANK       },
{   false,  TRACK_SLOPE_DOWN_25,    TRACK_BANK_NONE,    TRACK_CURVE_RIGHT_SMALL,        TRACK_SLOPE_NONE,       TRACK_BANK_RIGHT,   TRACK_ELEM_RIGHT_QUARTER_TURN_3_TILES_25_DEG_DOWN_TO_RIGHT_BANK     },
```

**Comment by Darkttd** [comment]
When i select large curve in same case, Vanilla has no option but, OpenRCT2 has option. (but if i select, there are no available track) is it data missed?

vanilla:
![banilla_scr1](https://cloud.githubusercontent.com/assets/6015765/14712948/305efe90-081a-11e6-8713-167b568eac4e.png)

OpenRCT2:
![scr2](https://cloud.githubusercontent.com/assets/6015765/14712952/364922a4-081a-11e6-9f8f-063833b46b8c.png)

**Comment by Darkttd** [comment]
This code is strange. when this case, disable condition is TRACK_CURVE_LEFT, not TRACK_CURVE_LEFT_SMALL. (same to right case)

`if (_currentTrackCurve != TRACK_CURVE_NONE) {
	// codes..
	if (_currentTrackSlopeEnd == _previousTrackSlopeEnd) {
		// codes..
		if (_currentTrackSlopeEnd == TRACK_SLOPE_DOWN_25) {
			disabledWidgets |= (1ULL << WIDX_SLOPE_DOWN_STEEP);
			if (_currentTrackCurve == TRACK_CURVE_LEFT_SMALL || _currentTrackCurve == TRACK_CURVE_RIGHT_SMALL || _rideConstructionState != RIDE_CONSTRUCTION_STATE_FRONT || !is_track_enabled(TRACK_SLOPE_CURVE_BANKED)) {
				disabledWidgets |= (1ULL << WIDX_LEVEL);
			}
		}
	// codes..
}`

**Comment by IntelOrca** [comment]
What version of vanilla RCT2 are you using?

The GOG version for example has a number of fixes over the original CD version (which is what was used as reference for implementing)

**Comment by Darkttd** [comment]
My version of vanilla RCT2 is Steam version.

**Comment by Gymnasiast** [comment]
Fixed by #3369.
